components/button/button.js

```diff
@@ -83,17 +90,19 @@ class Button extends ButtonMixin(LitElement) {
 				aria-label="${ifDefined(this.description || this.ariaLabel)}"
```

Comment:
So you decided to not apply `aria-disabled`?

Reply:
Just following up with Jeff about that.  This is what gets announced in VO and NVDA though.  Note the addition of "dimmed" in the VO case, and "unavailable" in the NVDA case.

VO: "[button text], dimmed, button." [tooltip text] You are currently on a button.  This item is dimmed.
NVDA: "clickable [button text] button unavailable [tooltip text]"

Reply:
How does that compare to:
1. A normal disabled button
2. The exact same setup but without `aria-disabled`

Reply:
1. A normal disabled button - nothing to compare to, since it's not possible to focus on a disabled button.

2. Same setup without `aria-disabled`

> VO: [button text], button. [tooltip text] You are currently on a button. To click this button, press Control-Option-Space.
NVDA: clickable [button text] button [tooltip text]

So yeah, without the `aria-disabled="true"`, VO keeps announcing "To click this button, press Control-Option-Space." like it would for an enabled button, which is definitely misleading.

I spoke with Jeff about the dimmed/unavailable this as well, and unless you disagree, I think we're all in favour of using `aria-disabled="true"`.

Reply:
> nothing to compare to, since it's not possible to focus on a disabled button

You must be able to access it though using arrow keys or by pulling up a list of buttons/form inputs? It's not invisible... just not focusable by tabbing through the page.

But yeah, I'm 100% in agreement that we want this -- I can't think of a single reason not to.

Reply:
To summarize:

Virtual cursor, normal disabled button:

>VO: [button text], dimmed, button. You are currently on a button. This item is dimmed.
>NVDA: [button text] selected

Virtual cursor, `aria-disabled` with tooltip:

>VO: [button text], dimmed, button. [tooltip text] You are currently on a button. This item is dimmed.
>NVDA: [button text] selected

Focusing, normal disabled button: N/A. 

Focusing, `aria-disabled` with tooltip:

>VO: [button text], dimmed, button. [tooltip text] You are currently on a button. This item is dimmed.
>NVDA: clickable [button text] button unavailable [tooltip text]

Focusing, regular button:

>VO: [button text], button. [tooltip text] You are currently on a button. To click this button, press Control-Option-Space.
>NVDA: clickable [button text] button [tooltip text]

Conclusion: use `aria-disabled="true"` for our simulated disabled state.  Of interest, NVDA only announces the tooltip text when focusing - it does not announce it when using the virtual cursor.

Reply:
Amazing -- agreed!